src/watertap_contrib/seto/unit_models/surrogate/lt_med_surrogate.py

```diff
@@ -759,21 +770,146 @@ def eq_cool_vol_flow(b):
                 to_units=pyunits.m**3 / pyunits.hr,
             )
 
+    def initialize_build(
+        blk,
+        state_args=None,
+        outlvl=idaeslog.NOTSET,
+        solver=None,
+        optarg=None,
+    ):
+        """
+        General wrapper for initialization routines
+
+        Keyword Arguments:
+            state_args : a dict of arguments to be passed to the property
+                         package(s) to provide an initial state for
+                         initialization (see documentation of the specific
+                         property package) (default = {}).
+            outlvl : sets output level of initialization routine
+            optarg : solver options dictionary object (default=None)
+            solver : str indicating which solver to use during
+                     initialization (default = None)
+
+        Returns: None
+        """
+        init_log = idaeslog.getInitLogger(blk.name, outlvl, tag="unit")
+        solve_log = idaeslog.getSolveLogger(blk.name, outlvl, tag="unit")
+
+        opt = get_solver(solver, optarg)
+        # ---------------------------------------------------------------------
+        flags = blk.feed_props.initialize(
+            outlvl=outlvl,
+            optarg=optarg,
+            solver=solver,
+            state_args=state_args,
+            hold_state=True,
+        )
+        init_log.info("Initialization Step 1a Complete.")
+        # ---------------------------------------------------------------------
+        # Initialize other state blocks
+        # Set state_args from inlet state
+
+        if state_args is None:
+            blk.state_args = state_args = {}
+            state_dict = blk.feed_props[
+                blk.flowsheet().config.time.first()
+            ].define_port_members()
+
+            for k in state_dict.keys():
+                if state_dict[k].is_indexed():
+                    state_args[k] = {}
+                    for m in state_dict[k].keys():
+                        state_args[k][m] = state_dict[k][m].value
+                else:
+                    state_args[k] = state_dict[k].value
+
+        state_args_dist = deepcopy(state_args)
+        for p, j in blk.distillate_props.phase_component_set:
+            if j == "TDS":
```

Comment:
Should we generalize this to check that component is Solute() rather than a hard-coded "TDS"?

Reply:
Actually I think the solute (and only solute) for this surrogate _has_ to be ``TDS``, so maybe we should put a check for that somewhere? This is a better question for @Zhuoran29

Reply:
Right, at this point, the surrogate applies to seawater, so TDS does make sense. I just thought it could be beneficial to keep it generic now in case there were ever an update to the model (even though I doubt it).